tests/userdashboard/rules/test_rule_change_moderation_comment.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import rules
+
+from adhocracy4.test.helpers import setup_users
+
+perm_name = "a4_candy_userdashboard.change_moderation_comment"
```

Comment:
Since we are [overwriting the rulesmethodmap](https://github.com/liqd/adhocracy-plus/blob/907c9ca11875ee99eaa6f35b88e940f852985e21/apps/userdashboard/api.py#L97) anyway, I think it would also work with just using a4.moderate_comment for both viewing and changing the comment. But this way I think it is more inline with how we do other permissions, no? Anyway, I dont mind changing it if you think its better!